GsaGH/Helpers/Assembly/ModelAssembly.cs

```diff
@@ -352,7 +352,8 @@ private void ConvertAndAssembleGridLines(List<GsaGridLine> gridLines) {
       if (gridLines != null) {
         int id = 1;
         foreach (GsaGridLine gridLine in gridLines.OrderBy(x => x.GridLine.Label)) {
-          _gridLines.SetValue(id, gridLine.GridLine);
+          GridLine apiGridLine = gridLine.GetApiGridLineToUnit(RhinoUnit.GetRhinoLengthUnit());
```

Comment:
already unit is stored in _ unit variable. Check How It has been used rather than getting rhinolengthunit?